lib/assert.js

```diff
@@ -118,10 +118,12 @@ x.doesNotThrow = function (fn, msg) {
 	try {
 		assert.doesNotThrow(fn, msg);
 	} catch (err) {
-		test(false, create(err.actual, err.expected, err.operator, err.message, x.doesNotThrow));
+		test(false, create(err.actual, err.expected, err.operator, err.message, x.notThrows));
 	}
 };
 
+x.doesNotThrow = x.notThrows;
```

Comment:
:+1: 
It would be good to start doing that to anything we think we want to drop with `1.0.0`